cfpb/cfgov-refresh
# Update conference form labels based on clearance

Updates copy in the conference registration form labels to match the latest cleared content. See `GHE/CFGOV/data-research/issues/421`

## Additions

- class for the required field note

## Removals

- helper text for the accommodations field

## Changes

- Copy for accommodations and food restrictions fields

## Testing

- check out locally
- Create a 'conference registration form' element in the Wagtail admin + add content required fields
- View page and confirm the copy on page matches screenshots here

## Review

- @chosak 
- @nataliafitzgerald 

## Screenshots

![screen shot 2016-11-21 at 3 09 00 pm](https://cloud.githubusercontent.com/assets/702526/20498835/ad170ba0-affc-11e6-865c-d4a0783111df.png)
![screen shot 2016-11-21 at 2 04 57 pm](https://cloud.githubusercontent.com/assets/702526/20496934/38f66c4a-aff5-11e6-8871-00d063bb34a9.png)

## Notes

-

## Todos

-

## Checklist
